source/client/benchmark_client_impl.h

```diff
@@ -123,6 +137,12 @@ class BenchmarkClientHttpImpl : public BenchmarkClient,
   StatisticPtr response_statistic_;
   StatisticPtr response_header_size_statistic_;
   StatisticPtr response_body_size_statistic_;
+  StatisticPtr latency_1xx_statistic_;
```

Comment:
Why do we list these here individually? Can we reuse the `struct BenchmarkClientStatistic` defined above?

Reply:
Yes, changed to reuse BenchmarkClientStatistic struct here.